src/style.css

```diff
@@ -1 +1,139 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+}
```

Comment:
This one doesn't work in tests. Use the link in HTML.  You don't need to download files with fonts.

https://fonts.google.com/specimen/Roboto?query=roboto